traffic_stats/influxdb_tools/create/create_ts_databases.go

```diff
@@ -20,45 +20,81 @@ under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
-	"os"
 
 	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
```

Comment:
So as far as the [errors](https://github.com/pkg/errors) package is concerned, it's mostly to contextualize errors without doing something like 
```go
if err != nil {
   errorMessage = fmt.Sprintf("Some string %v", record) 
   fmt.Println(errorMessage)
}
```

This pattern was recurring in the code, and not only was it needlessly verbose (and using the global var errorMessage), but it also loses the original error, so debugging would entail going back to the code and adding Printf debugging, before the error itself could actually be addressed. The errors package functionality is actually quite simple, but like most simple code, it is very effective in what it does. 

The [cli](https://github.com/urfave/cli) package makes clis generally more user friendly, but also has some additional functionality (which I didn't add to this PR) of being able to set values on pointer receivers, declaring shell environment variables (helpful with containerized/orchestrated environment deployments).

As I was going through the codebase I was trying to come up with the different ways that it could be made more "idiomatic" (which I admit is a very vague term). Currently, some of the code seems more C/script-ish, and doesn't seems to take much advantage of the language beyond the syntax and garbage collecting.  This could well be because of constraints specified elsewhere, but if so, I didn't see anything that made me aware of them.